bin/cli-runner.js

```diff
@@ -16,6 +16,10 @@ module.exports.run = args => {
     .version()
     .help();
   const cmdPath = path.resolve(cmdsDir, `${instance.argv._}.js`);
+  // we should always have process.cwd() be equal to the dir of the project being run
```

Comment:
I'm not so sure about this. I think `process.cwd()` should only used as part of a default for `--dir`. Otherwise, I think `process.cwd()` should never be referenced anywhere in the CLI.

Reply:
I think that it helps a lot when dealing with using the file system. I don't see much of a downside to doing it, and the upside of making filesystem concerns consistent. What are the downsides that you see?